### PR TITLE
Disable `01600_parts_types_metrics_long` for asan

### PR DESCRIPTION
**Pull request by antonio2368**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

The test is verifying that the counts in `system.parts` and number in `system.metrics` are the same but it checks for all tables because we cannot filter `system.metrics` for specific table. 
So this test just verifies that **sometimes** those counts match.
We rely heavily on luck while basically not proving anything, especially when running in parallel where a lot of parts are created and removed. 

cc @azat as the author of the test, you can correct me if I'm wrong

### Documentation entry for user-facing changes

- [ ] Documentation is written (mandatory for new features)

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

### CODE REVIEW

**Comment by robot-ch-test-poll2** [comment]
This is an automated comment for commit d9ac7e12cb138bfddad1f68f073568ac00daa7f4 with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/54132/d9ac7e12cb138bfddad1f68f073568ac00daa7f4/ci_running.html)
The overall status of the commit is 🟡 pending

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td>🟡 pending</td></tr>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td>🟢 success</td></tr>
</table>

File: tests/queries/0_stateless/01600_parts_types_metrics_long.sh [hunk 1]
```diff
@@ -1,49 +0,0 @@
-#!/usr/bin/env bash
-# Tags: no-s3-storage
```
Comment (azat) [inline tests/queries/0_stateless/01600_parts_types_metrics_long.sh:2]: Maybe instead mark it as `no-parallel`?

Comment (antonio2368) [reply tests/queries/0_stateless/01600_parts_types_metrics_long.sh:2]: there are still system tables creating parts, right?

Comment (azat) [reply tests/queries/0_stateless/01600_parts_types_metrics_long.sh:2]: Yes, but there are retries in the test, `no-parallel` + limited number of retries should help I guess.

Comment (antonio2368) [reply tests/queries/0_stateless/01600_parts_types_metrics_long.sh:2]: let's skip it for asan for now as those are the only failures I see for it
if it doesn't help we will see what to do next

**Review by alexey-milovidov — APPROVED** [review]

**Pull request merged**